oauthproxy.go

```diff
@@ -729,14 +743,12 @@ func (p *OAuthProxy) UserInfo(rw http.ResponseWriter, req *http.Request) {
 
 // SignOut sends a response to clear the authentication cookie
 func (p *OAuthProxy) SignOut(rw http.ResponseWriter, req *http.Request) {
-	redirect, err := p.GetRedirect(req)
```

Comment:
- put COCO Begin here, as the entire function is changed by us

Reply:
handled